File: test/test_input.rb
puts "0:#{$0}"
puts "1:#{ARGV[0]}"
puts "2:#{ARGV[1]}"
File.open('/dev/tty', 'r') do |device|
  puts "input data:"
  data = device.gets.chomp
  puts "data is #{data} "
end

File: test/test_input.py
import sys
print("0:"+sys.argv[0])
print("1:"+sys.argv[1])
print("2:"+sys.argv[2])
with open('/dev/tty', 'r') as device:
    print("input data:")
    data = device.readline()
    print("data is "+str(data))
